README.md

```diff
@@ -168,35 +178,35 @@ The available options also change in accordance with the `inline` value. With th
 
 ```js
 // If false or not sent, options can be sent like other plugins
-[withSVG({
+withSVG({
     options: {
         limit: 20000,    // will be safely passed to url-loader
     },
-})],
+}),
 
 // If sent true, 'use' value will override default loaders entirely
-[withSVG({
-    exclude: /inline\/.*.svg$/,    // Will be safely passed to rule
+withSVG({
+    include: /\.inline\./,        // Will be safely passed to rule
     inline: true,
     use: [{
         loader: 'url-loader',     // Only 'url-loader' will be used
     }],
-})],
+}),
 ```
 
 The following example shows how you can use the inline option in your project:
 
 ```js
-// Inline SVGs are stored in /inline/
-[withSVG({
-    include: /inline\/.*.svg$/,
+// Inline SVGs have a '.inline' suffix
```

Comment:
Rephrase to `Inline SVGs that have a ...` or (imo better) use `Include SVGs with '.inline' suffix`.